### ISSUE TITLE
Error! Access is forbidden, or API overused.

### ISSUE THREAD

**Issue by telotortium** [optroot/watch-later#9]
For several days now I've gotten this popup 100% of the time when trying to use this extension:

> Error!
> Access is forbidden, or API overused.

If I look at the network traffic in Chrome DevTools, I see the following response to POST requests to https://www.googleapis.com/youtube/v3/playlistItems?part=snippet with body `{"snippet":{"playlistId":"WL","resourceId":{"kind":"youtube#video","videoId":"[redacted]"}}}`:

```
{
  "error": {
    "code": 403,
    "message": "The request cannot be completed because you have exceeded your \u003ca href=\"/youtube/v3/getting-started#quota\"\u003equota\u003c/a\u003e.",
    "errors": [
      {
        "message": "The request cannot be completed because you have exceeded your \u003ca href=\"/youtube/v3/getting-started#quota\"\u003equota\u003c/a\u003e.",
        "domain": "youtube.quota",
        "reason": "quotaExceeded"
      }
    ]
  }
}
```

Perhaps the project has reached a project-wide quota limit? Looks similar to #7.

**Comment by kosakk** [comment]
I am experiencing this same error also. 

![2020-06-12-195252_349x70_scrot](https://user-images.githubusercontent.com/12464123/84526804-83170180-ace6-11ea-9842-dfc27c5b1cf6.png)

**Comment by telotortium** [comment]
Yes, my diagnosis is correct. So, unless the original author wants to pay to enable more API requests (unlikely), you'll need to clone this Github repo to get a local copy of the extension, use "load unpacked extension" to install it, and follow https://developer.chrome.com/extensions/tut_oauth to create a new OAuth app for your personal use, which will enable you to get around the API limits (assuming you don't use it an absurd number of times per day). Sorry :(

P.S. Make sure to opt-out of the new Chrome Developer Console (bottom left corner of the new developer console when you open it). Otherwise, you won't be able to find your extension's full public key except by searching through DevTools.

**Comment by GeoffreyPlitt** [comment]
I wish the original author accepted micropayments

**Comment by GeoffreyPlitt** [comment]
@telotortium Having trouble following your advice.
- I did not see any way to opt-out of the new Chrome Developer Console-- didn't see anything like you describe in a bottom left corner. I can't tell if I'm in the new one or the old one, because I haven't used it before.
- I was able to upload a ZIP in draft mode. But then according to https://developer.chrome.com/extensions/tut_oauth, it says "Find the extension under Your Listings and click on more info."  I see my extension, but there's no "more info" anywhere on the screen.  Clicked every menu, explored everything I could find about my draft extension, didn't find it. Any ideas?

Screenshot:
<img width="1033" alt="Screenshot 2020-06-17 19 22 35" src="https://user-images.githubusercontent.com/382821/84970621-eda5b400-b0cf-11ea-9f74-f222ca2c43c9.png">

**Comment by ElmorenohWTF** [comment]
The extension isn't working for me too and I don't know how to do what you are saying, do I have to pay something to do it? Because I don't want to, but I need a extension like this.

**Comment by GeoffreyPlitt** [comment]
@optroot @telotortium Any guidance? The instructions above don't work on the Chrome developer console.

**Comment by telotortium** [comment]
Look carefully for the old console link - it's there on the left-hand
corner. If you can't find it, you should be able to use Devtools and search
for it in the DOM (i.e., HTML).

**Comment by GeoffreyPlitt** [comment]
@telotortium Thank you for your help. I searched in the HTML and didn't see any obvious links to the old console. Every link with the word "console" in it went to the new console. I guess I don't know what I'm looking for... What's the URL pattern?

**Comment by telotortium** [comment]
Ah, you need to click "Show More" in the bottom left corner, which will expand to the link to the old console ("opt out").

![Screen Shot 2020-06-25 at 12 11 15](https://user-images.githubusercontent.com/1755789/85784580-46330d80-b6dd-11ea-9d62-38a15368e315.png)

**Comment by GeoffreyPlitt** [comment]
@telotortium Thanks! That worked. Some other questions.

Which scopes are needed? I took a guess at "https://www.googleapis.com/auth/youtube" and that seemed to trigger the right youtube auth.

But then once auth succeeded, I went to use it (clicked watch-later on a youtube link) and nothing happened. No JS console errors. How to troubleshoot?

**Comment by ElmorenohWTF** [comment]
If someone gets it to work, please share the extension with me, I can't pay the 5$ to register
